app/javascript/pages/forgot-password.tsx

```diff
@@ -1,19 +1,59 @@
-import React from "react"
-
-import { t } from "@bloom-housing/ui-components"
-
-import withAppSetup from "../layouts/withAppSetup"
-import { Heading } from "@bloom-housing/ui-seeds"
+import React, { useState } from "react"
+import { AppearanceStyleType, Button, Form, t, FormCard, Icon } from "@bloom-housing/ui-components"
+import { Link, Heading } from "@bloom-housing/ui-seeds"
+import { useForm } from "react-hook-form"
+import { getSignInPath } from "../util/routeUtil"
+import EmailFieldset from "../pages/account/components/EmailFieldset"
+import "../pages/account/styles/account.scss"
 import FormsLayout from "../layouts/FormLayout"
+import withAppSetup from "../layouts/withAppSetup"
 
-interface ForgotPasswordProps {
-  assetPaths: unknown
+const ForgotPassword = () => {
+  const EmailSubmittedCard = () => {
+    return (
+      <div className="text-center p-8">
+        <Heading priority={3} size="2xl" className="mb-4">
+          {t("forgotPassword.emailSent")}
+        </Heading>
+        <p>{t("forgotPassword.emailLink")}</p>
+      </div>
+    )
+  }
+  const [emailSubmitted, setEmailSubmitted] = useState(true)
+  // eslint-disable-next-line @typescript-eslint/unbound-method
+  const { register, handleSubmit, errors } = useForm()
+  const onSubmit = (data: { email: string }) => {
+    setEmailSubmitted(false)
```

Comment:
**question (non-blocking):**  The way I am reading this is that emailSubmitted should start as false, and then once the form is submitted, we want to show the correct card. Am I understanding that correctly?